youtube-dl-server.py

```diff
@@ -123,18 +123,20 @@ def dl_worker():
 
 
 def build_youtube_dl_cmd(url):
-    if (url[2] == "best"):
-        cmd = ["youtube-dl", "--proxy", proxy, "-o", "./downfolder/.incomplete/%(title)s.%(ext)s", "-f", "bestvideo[ext=mp4]+bestaudio[ext=m4a]/best[ext=mp4]", "--exec", "touch {} && mv {} ./downfolder/", "--merge-output-format", "mp4", url[0]]
-    # url[2] == "audio" for download_rest()
-    elif (url[2] == "audio-m4a" or url[2] == "audio"):
-        cmd = ["youtube-dl", "--proxy", proxy, "-o", "./downfolder/.incomplete/%(title)s.%(ext)s", "-f", "bestaudio[ext=m4a]", "--exec", "touch {} && mv {} ./downfolder/", url[0]]
-    elif (url[2] == "audio-mp3"):
-        cmd = ["youtube-dl", "--proxy", proxy, "-o", "./downfolder/.incomplete/%(title)s.%(ext)s", "-f", "bestaudio[ext=m4a]", "-x", "--audio-format", "mp3", "--exec", "touch {} && mv {} ./downfolder/", url[0]]
-    else:
-        resolution = url[2][:-1]
-        cmd = ["youtube-dl", "--proxy", proxy, "-o", "./downfolder/.incomplete/%(title)s.%(ext)s", "-f", "bestvideo[height<="+resolution+"][ext=mp4]+bestaudio[ext=m4a]", "--exec", "touch {} && mv {} ./downfolder/",  url[0]]
-    print (" ".join(cmd))
-    return cmd
+    with open('Auth.json') as data_file:
```

Comment:
@Hervey424 Could you tell me the location of this `Auth.json` file on disk, its expected content, and who or which code is supposed to create it?

Reply:
Opps, sorry, found it here https://github.com/hyeonsangjeon/youtube-dl-nas/blob/1c3318a259879c2e709c70ee0d7a0368fa4dcfd6/Auth.json